### ISSUE TITLE
About pretraining in ULIP2.

### ISSUE THREAD

**Issue by xiatiandefeng666** [OpenRobotLab/PointLLM#49]
When conducting pretraining based on Point-BERT in ULIP2, did you use text-point cloud pairs from Cap3D, or multi-view image-to-top-text pairs along with point clouds? Thank you very much！

**Comment by RunsenXu** [comment]
multi-view + text + point cloud

**Comment by xiatiandefeng666** [comment]
Thank you！ If I use ULIP-2, does it support pre-training of  only point cloud + 3D text?

**Comment by RunsenXu** [comment]
Yes you can remove the loss for the image branch.
